hpx/runtime/applier/apply.hpp

```diff
@@ -444,7 +444,6 @@ namespace hpx
 #ifdef HPX_HAVE_APEX
             data.apex_data = hpx::util::apex_new_task(
                 data.description,
-                data.parent_locality_id,
                 data.parent_id);
```

Comment:
The idea of passing the `parent_locality_id` to APEX was to enable cross-locality tracking of things. Could you elaborate on the rationale of removing this, please?

Reply:
Well, we might've been too eager with this, but my reasoning was that the argument is unused. If it is to be made to work (again?) it can be put back in.

Reply:
This might have been in the planning all the time. I'd suggest to leave it in for the time being, for this reason.

Reply:
They're back.